content/pages/how_you_can_use_python.md

```diff
@@ -0,0 +1,29 @@
+Title: {{ How You Can Use Python }}
+Content Date: 2018-10-23 22:27
+
+Tags: python, learning, programming, projects, ideas
+
+Authors: Cleveland PyLadies, Daniel Medina
+
+Summary:  A brief summary of how you can use and apply Python to solve real-world problems.
+
+Python is an extremely versatile programming language known for its beginner friendly syntax.  It is used by NASA, Google, Dropbox, Quora, and many others.
+
+Python is primarily used for scripting, the automation of short repetitive tasks.
+
+Here are some of the ways that Python is used across multiple areas:
+
+  Science with [SciPy](https://www.datacamp.com/community/tutorials/python-scipy-tutorial), [NumPy](https://www.machinelearningplus.com/python/numpy-tutorial-part1-array-python-examples/)
+
+  Web Development with [Django](https://simpleisbetterthancomplex.com/series/2017/09/04/a-complete-beginners-guide-to-django-part-1.html) and [Flask](https://www.tutorialspoint.com/flask/)
+
+  Machine Learning with [Scikit-learn](https://medium.com/@mwitiderrick/introduction-to-supervised-learning-with-python-scikit-learn-tutorial-966457dd33b9)
```

Comment:
`* Machine Learning...` (omg MORE. There are [so many ML libraries](https://towardsdatascience.com/best-python-libraries-for-machine-learning-and-data-science-part-1-f18242424c38)! There will be a LOT of cross-over in ML/data/science and that's ok. Repeating the libraries conveys the reality that many of these libraries are used in multiple domains.)